### DISCUSSION TITLE
[✨]SSG Ignore Authenticated Routes Without Warning

### DISCUSSION THREAD

**Discussion by ToanTrinh01** [QwikDev/qwik-evolution#110] [Proposals For Qwik]
### Is your feature request related to a problem?

When running static adapter, I need a list of every page that was generated and every page that was not generated

### Describe the solution you'd like

When running static adapter I need a list of any page that was not generated so that it could help me debug the problem

### Describe alternatives you've considered

Adding a env variable file in order to specify which page should be generated and bypass authentication

### Additional context

I may want authentication middleware to handle by my hosting provider

It would still be a good idea to know which page is not being generated

**Comment by gioboa** [comment]
Hi, thanks for your suggestions.
Correct me if I'm wrong but in this issue you mentioned different enhancements, right?
Can you create a list of these features pls? because right now it's hard to understand.

**Comment by PatrickJS** [comment]
we need a list of routes generated and not generated

**Comment by gioboa** [comment]
We moved this issue to `qwik-evolution` repo to create a RFC discussion for this.
Here is our [Qwik RFC process](https://github.com/QwikDev/qwik-evolution?tab=readme-ov-file#qwik-rfc-process) thanks.
